core/templates/domain/collection/collection-node.model.spec.ts

```diff
@@ -71,4 +71,91 @@ describe('Collection node model', () => {
       expect(collectionNode.doesExplorationExist()).toBe(false);
     }
   );
+
+  it('should be able to detect if exploration is private',
+    () => {
+      var explorationSummaryBackendObject = {
```

Comment:
Add the `explorationSummaryBackendObject` type

Reply:
Done!